src/components/Modal.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const rootModal = document.querySelector('#modal');
```

Comment:
```suggestion
const rootModal = document.querySelector('#modal-root');
```

Reply:
Actually it's supposed to be just `#modal`, because when you create the portal you append a child to it:
https://reactjs.org/docs/portals.html

<img width="542" alt="Screen Shot 2019-09-05 at 14 28 26" src="https://user-images.githubusercontent.com/54307999/64338081-7b089380-cfe9-11e9-81b8-0a561c302f00.png">